### ISSUE TITLE
AssertionConsumerServiceUrl is always null

### ISSUE THREAD

**Issue by EugeneZhudro** [ITfoxtec/ITfoxtec.Identity.Saml2#69]
Hi           

When we read AssertionConsumerServiceUrl  in Saml2AuthnRequest it  read from xml element

https://github.com/ITfoxtec/ITfoxtec.Identity.Saml2/blob/d3e990d5b2f59805b437f7e5f6c9f1bac3bb0512/src/ITfoxtec.Identity.Saml2/Request/Saml2AuthnRequest.cs#L137

In result AssertionConsumerServiceUrl is always null because AssertionConsumerServiceUrl  is an attribute in AuthNRequest

Thanks,
Eugene

**Comment by dmpe** [comment]
Can actually, I think, confirm this. also getting just this, instead of having 2 additional fields "AssertionConsumerServiceUrl" and bindings
```
<saml2p:AuthnRequest xmlns:saml2p="urn:oasis:names:tc:SAML:2.0:protocol"
                     xmlns:saml2="urn:oasis:names:tc:SAML:2.0:assertion"
                     ID="_6e88ef2b-1829-43ae-a0ad-3e60d4f95f8e"
                     Version="2.0"
                     IssueInstant="2021-02-24T12:48:08.1195378Z"
                     Destination="xxxxx"
                     >
````

@Revsgaard any ideas ?

**Comment by Revsgaard** [comment]
Thank you for contributing. Your pull request is released in version 4.3.1 https://github.com/ITfoxtec/ITfoxtec.Identity.Saml2/releases/tag/4.3.1.
